spec/factories/kalibro_configuration_attributes.rb

```diff
@@ -0,0 +1,14 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :kalibro_configuration_attributes, class: KalibroConfigurationAttributes do
+    sequence(:id, 1)
+    association :user, strategy: :build
```

Comment:
It creates an instance of the `:user` factory by default, in an attribute called `user`, if you don't pass an `:user => something` parameter to `FactoryGirl.build` or `FactoryGirl.create`. It's just a shorthand.